katas/metal-slug/mateo/vehicle.js

```diff
@@ -0,0 +1,5 @@
+export class Vehicle {
+    constructor(aditionalHealth) {
+        this.aditionalHealth = aditionalHealth;
```

Comment:
Why does vehicle has `additionalHealth`? couldn't the attribute just be `health`? I think "additional" health can make sense from the point of view of the soldier when he/she gets into the vehicle, but soldier is not actually part of this class directly, so we can handle that concept independently